quinn-proto/src/transport_error.rs

```diff
@@ -49,7 +49,7 @@ pub struct Code(u64);
 
 impl Code {
     /// Create QUIC error code from TLS alert code
-    pub(crate) fn crypto(code: u8) -> Self {
+    pub fn crypto(code: u8) -> Self {
```

Comment:
Not sure how you want to handle this one. If we want to be picky, we could make this a `try_from` pattern, as is done elsewhere in the code.

Reply:
I think this is infallible, and suitable for exposure as-is. From the RFC:

> CRYPTO_ERROR (0x0100-0x01ff):
>    The cryptographic handshake failed. A range of 256 values is reserved for carrying error codes specific to the cryptographic handshake that is used. Codes for errors occurring when TLS is used for the cryptographic handshake are described in [Section 4.8](https://www.rfc-editor.org/rfc/rfc9001#section-4.8) of [[QUIC-TLS](https://www.rfc-editor.org/rfc/rfc9000.html#QUIC-TLS)].

Note in particular that the cryptographic layer is free to define these as it sees fit.